### ISSUE TITLE
Text/Background Color contrast too low 

### ISSUE THREAD

**Issue by saad-mo** [SAP/spartacus#9360]
**Describe the bug**
Text/Background Color contrast too low 

**To Reproduce**
1. log in as an approver
2. Go to approval dashboard 
3. Observe the pending approvals table header 

**Expected behavior**
The contrast should be higher as this is an accessibility issue

**Screenshots**
<img width="1144" alt="Screen Shot 2020-10-13 at 4 14 12 PM" src="https://user-images.githubusercontent.com/15215400/96159907-dc807e80-0eda-11eb-9eee-dd5e9ca58c48.png">

**Desktop (please complete the following information):**
 - OS: iOS
 - Browser: Chrome
 - Version:  85.0.4183.121

**Comment by developpeurweb** [comment]
@Manurod2018 you may notice that the table header doesn't follow the styles we use on Spartacus tables (no background colors) and it's currently very low contrast. 
Can we please get a designer's input on this?

**Comment by Manurod2018** [comment]
Change the font color to white for now. 
@saad-mo this looks like it's an old screen, please refer to this Zeplin project for the new styling: https://zpl.io/2ENnmAY

**Comment by developpeurweb** [comment]
Text is now set to white as per screenshots

![Screen Shot 2020-10-19 at 2 59 30 PM](https://user-images.githubusercontent.com/15113219/96502471-d57da700-121f-11eb-975d-081157ae86dc.png)
![Screen Shot 2020-10-19 at 3 00 56 PM](https://user-images.githubusercontent.com/15113219/96502473-d6163d80-121f-11eb-8f51-9e7d2d094f11.png)

**Comment by saad-mo** [comment]
> Change the font color to white for now.
> @saad-mo this looks like it's an old screen, please refer to this Zeplin project for the new styling: https://zpl.io/2ENnmAY

I do not have access to the project. Could you please add me? (saad.mohiuddin@sap.com / saadm)

**Comment by saad-mo** [comment]
Performed QA on the latest develop on local and works as expected. 

<img width="1056" alt="Screen Shot 2020-10-21 at 11 01 57 AM" src="https://user-images.githubusercontent.com/15215400/96746595-f6173f80-138c-11eb-9e74-8995cc3cc701.png">
